fdbsyncd/fdbsyncd.cpp

```diff
@@ -44,8 +46,28 @@ int main(int argc, char **argv)
              */
             if (sync.getRestartAssist()->isWarmStartInProgress())
             {
+                int pasttime = 0;                

```

Comment:
use steady_clock ?

Reply:
sure .. will follow same logic as used in neighborsyncd